src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -874,7 +874,17 @@ ServerHandlerImp::processRequest(
             {user, forwardedFor}};
         Json::Value result;
         auto start = std::chrono::system_clock::now();
-        RPC::doCommand(context, result);
+        try
+        {
+            RPC::doCommand(context, result);
+        }
+        catch (std::exception const& ex)
+        {
+            result[jss::result] = RPC::make_error(rpcINTERNAL);
```

Comment:
Is this nesting correct? Or are we supposed to just assign to `result` here? @natenichols

Reply:
I _think_ we just assign to `result` here. 

The code at Line 901/902 and just uses `result[jss::error_code]` and `result[jss::error]`.

It looks like that code path would get run here, and those values would be `null`.